src/index.js

```diff
@@ -1,10 +1,6 @@
-import Bar from './Bar';
-import Pie from './Pie';
-import Line from './Line';
-import XY from './XY';
-import Radar from './Radar';
-import config from './config';
-
-module.exports = {
```

Comment:
@liady I think, this would pose the inconsistency in the codebase, most of the files have exports at end file. 
@timqian Any thoughts?

Reply:
Yeah, consistency is important. And personally I prefer doing  exports at the end of a file because it is easier to tell what a file is exporting (by looking at the bottom)